src/sass/letters.scss

```diff
@@ -67,6 +65,15 @@
     }
   }
 
+  .va-introtext {
```

Comment:
There's a lot of re-organizing and reformatting--is chunk all that's new in this PR? `.va-introtext` and `.progress-bar-segmented` with the use of `schemaform-padding`.

Reply:
Er yeah, sorry. Linter didn't like the way things were organized so I just re-shuffled some stuff that should have probably just been done in a separate PR. The only substantive changes here are the removal of `.step-content`, `.letters-progress-bar`, and addition of the media-query mixin to `.progress-bar-segmented` as well as `.va-intro-text`